app/views/categories/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="categories--container">
+  <article class='form--header'>
+    <h2>Categories</h2>
+    <%= link_to 'LOG OUT', destroy_user_session_path, data: {turbo_method: :delete} %>
+
+  </article>
+  <article class="catergories--list--container">
+    <ul class="categories--list">
+      <% @categories.each do |category| %>
+        <li class="categories--item">
+        <div class="categories--item--info">
+          <% if category.icon.attached? %>
+          <%= image_tag category.icon %>
+          <% else %>
+            <%= image_tag "https://media.designrush.com/inspiration_images/134933/conversions/_1511456189_555_McDonald's-preview.jpg" %>
+          <% end %>
+          <%= link_to category.name, category_expenses_path(category) %>
+        </div>
+        <p class="categories--item--price">$<%= format_amount(category.expenses.sum(:amount)) %></p>
+      </li>
+      <% end %>
+    </ul>
+  </article>
+
+  <article class="categories--btn--container">
+    <%= link_to 'Add a new category', new_category_path, class: "add--btn" %>
+  </article>
```

Comment:
- [ ] As required for the project your app to follow the the [the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from Behance, kindliy ensure that you update your current design by using the typography(font, size, weight,...), text-transformation, Layout(alignment and position of elements), colors, etc.

| Template design | your Design | 
| ---------------------- | -------------------- |
| ![Screenshot from 2023-11-21 17-22-34](https://github.com/Akorede3133/expense-insight/assets/83042742/fa478296-2a85-4de3-9f58-baa024a17dac) | ![ksnip_20231121-173100](https://github.com/Akorede3133/expense-insight/assets/83042742/ac63368b-0a18-4588-a79e-f179ed584540) |